test/unit/checkout.js

```diff
@@ -1,11 +1,55 @@
 import Checkout from '../../src/components/checkout';
 
 describe('Checkout class', () => {
-  describe('get params', () => {
-    it('puts together a big param string', () => {
-      const checkout = new Checkout({window: { height: 100, width: 100}});
-      const windowParams = 'height=100,width=100,';
-      assert.include(checkout.params, windowParams);
+  let checkout;
+  const config = {
+    window: {
+      height: 100,
+      width: 100,
+    },
+  };
+
+  beforeEach(() => {
+    checkout = new Checkout(config);
+  });
+
+  describe('constructor', () => {
+    it('sets config to config param', () => {
+      assert.equal(checkout.config, config);
+    });
+  });
+
+  describe('prototype methods', () => {
+    describe('open', () => {
+      it('opens window to checkout if cart is popup', () => {
+        const openStub = sinon.stub(window, 'open');
+        checkout.config.cart = {
+          popup: true,
+        };
+        checkout.open('test.com');
+        assert.calledOnce(openStub);
+        assert.calledWith(openStub, 'test.com', 'checkout', checkout.params);
+      });
+    });
+
+    describe('getters', () => {
+      describe('params', () => {
+        it('creates a string including window config params', () => {
```

Comment:
Maybe update getter test names to `it('returns...`, for consistency with other test files